EXTCAP.md

```diff
@@ -0,0 +1,195 @@
+
+### Description
+
+```
+    -h, --help            
+    	Show this help message and exit
+
+    --extcap-interfaces
+       	Provide a list of interfaces to capture from.
+
+    --extcap-interface <EXTCAP_INTERFACE>
+        Provide the interface to capture from.
+    
+    --extcap-dlts
+        Provide a list of dlts for the given interface
+
+    --extcap-config
+        Provide a list of configurations for the given interface.
+
+    --fifo <FIFO>
+        Use together with capture to provide the fifo to dump data to.
+    
+    --channel <CHANNEL>
+         IEEE 802.15.4 capture channel [11-26].
+
+    --baudrate <BAUDRATE>
+        Set the serial port baud rate.
+
+    --tap
+        Use to specify DLTs as IEEE 802.15.4 TAP (only for Wireshark3.0 and later).
+```
+
+## Quick start
+
+### Configuring Wireshark for Thread
+
+* Wireshark configuration - [Protocols](https://openthread.io/guides/ncp/sniffer#wireshark_configuration_-_protocols)
+* Wireshark configuration - [FCS Format](https://openthread.io/guides/ncp/sniffer#wireshark_configuration_-_rssi)
+    * For Wireshark 2.x: TI CC24xx metadata
+    * For Wireshark 3.x: ITU-T CRC-16
+   
+### Using the sniffer
+
+#### Wireshark welcome window
+
+The Wireshark welcome window is displayed when Wireshark is first launched. 
+OpenThread Sniffer will be enumerated in the interface list section of Wireshark welcome window.
+
+#### Start sniffing
+
+There are three ways to start sniffing:
+* If this is your first time using an interface, click on **Interface Options** 
+  to set channel and baudrate, then click **Start Capture**. 
+  The parameters will be saved after the start of the capture, and you will not
+  need to set it again the next time you use the interface (unless you need to
+  change the channel)
+
+* Double click on the hardware interface
+
+* Select the hardware interface and then click **Start Capture** to start sniffing.
+ 
+#### Capture from multiple hardware interfaces/boards
+ 
+ Select all hardware interfaces in the Wireshark welcome window and click 
+ the Wireshark icon on the top left to start sniffing.
+ 
+**Interface name (frame.interface_name)**\
+Interface Identifier used by Wireshark to identify the capture interfaces 
+
+**Channel (wpan-tap.ch_num)**
+
+## Troubleshooting
+
+### The OpenThread sniffer is not listed in the Wireshark interface. 
+
+1. If you have multiple Python interpreters installed, ensure that Python3 interpreter is used by the extcap script. Python2 is no longer supported.
+2. See if the hardware has been enumerated on USB and the drivers are loaded. 
+3. Check that the HEX file for the hardware has been flashed. 
+4. Reset the hardware by unplugging the hardware, waiting 5 seconds, and plugging it back in. 
+5. Restart Wireshark. If it still doesn’t appear, verify the python script located in the extcap folder is able to run. 
+    
+    For OS X and Linux:
+    1. Verify that the execute permission is present for the extcap_ot.py file. 
```

Comment:
"...execute (x) permission..." 

for consistency with the earlier instructions on this